qiskit/tools/backends/reduced_cmap.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Reduced coupling_map
+"""
+
+import numpy as np
+import scipy.sparse as sp
+import scipy.sparse.csgraph as cs
+from qiskit.qiskiterror import QiskitError
+
+
+def reduced_coupling_map(backend, mapping):
+    """Returns a reduced coupling map that
+    corresponds to the subgraph of backend qubits
+    selected in the mapping.
+
+    Args:
+        backend (BaseBackend): A backend for a physical device.
+        mapping (list): A mapping of reduced qubits to device
```

Comment:
I'm not sure what is this argument or how a list is a mapping....

Reply:
It is a mapping between the reduced coupling map qubits and the device qubits.